### PR TITLE
Revert "B 20256 main prevent prime users from accessing pptas"

### PR DESCRIPTION
**Pull request by pambecker**
Reverts transcom/mymove#13415
Tests for PPTAS not working in staging, need to revert this code.
I will create a PR for B-19181 after this one goes in.

### CODE REVIEW

**Review by paulstonebraker — APPROVED** [review]

File: migrations/app/migrations_manifest.txt
Comment (paulstonebraker) [inline migrations/app/migrations_manifest.txt:1]: should we not remove migrations?

Comment (deandreJones) [reply migrations/app/migrations_manifest.txt:1]: good call, there should be a script added, to remove the column in the client_certs table

Comment (pambecker) [reply migrations/app/migrations_manifest.txt:1]: So add this line back in and the actual 20240607134224_allow_pptas_client.up.sql file?
And create a new entry to delete the column, correct?

Comment (pambecker) [reply migrations/app/migrations_manifest.txt:1]: udpated

**Review by deandreJones — APPROVED** [review]

File: migrations/app/migrations_manifest.txt
Comment (deandreJones) [reply migrations/app/migrations_manifest.txt:1]: yea 1 tricky thing with db migrations and reverts.. we should not revert the DB stuff, but rather just add a script to reverse the changes

**Pull request merged**